### PR TITLE
"lakefs setup" to set up ACL groups

### PR DESCRIPTION
**Pull request by arielshaqed**
Depending on auth.ui_config.rbac "simplified" or "external", create
appropriate initial groups.

### CODE REVIEW

File: esti/auth_test.go [hunk 1]
```diff
@@ -66,134 +54,134 @@ func TestAdminPolicies(t *testing.T) {
...
+// Test Super Permissions: AuthManageOwnCredentials, FSFullAccess, RepoManagementReadAll
+func TestSuperPermissions(t *testing.T) {
 	ctx, logger, repo := setupTest(t)
-	gid := "SuperUsers"
+	gid := "Supers"
 
-	// generate the SuperUser client
-	superUserClient := newClientFromGroup(t, ctx, logger, gid)
+	// generate the Super client
+	superClient := newClientFromGroup(t, ctx, logger, gid)
```
Comment (guy-har) [inline esti/auth_test.go:63]: nit: no value in `gid` IMO just call `superClient := newClientFromGroup(t, ctx, logger, "Supers")`
Same for the following cases

**Review by guy-har — APPROVED** [review]
LGTM

File: esti/auth_test.go (hunk 1 above)
Comment (arielshaqed) [reply esti/auth_test.go:63]: Done!

File: pkg/auth/base.go [hunk 2]
```diff
@@ -0,0 +1,121 @@
+package auth
+
+import (
+	"github.com/treeverse/lakefs/pkg/auth/model"
+	"github.com/treeverse/lakefs/pkg/permissions"
+
+	"errors"
+	"fmt"
```
[the remark this answers is not in the document]
Comment (arielshaqed) [reply pkg/auth/base.go:8]: Done!

File: pkg/auth/setup/setup.go [hunk 3]
```diff
@@ -258,19 +152,54 @@ func SetupBaseGroups(ctx context.Context, authService Service, ts time.Time) err
...
+		return fmt.Errorf("create base ACL group %s: %w", acl.ACLReadersGroup, err)
+	}
+	if err := acl.WriteGroupACL(ctx, authService, acl.ACLReadersGroup, model.ACL{Permission: acl.ACLRead, Repositories: all}, ts, true); err != nil {
+		return fmt.Errorf("setup: %w", err)
+	}
+
+	return nil
+}
+
+// BUG(ariels): read config, do it right!
```
[the remark this answers is not in the document]
Comment (arielshaqed) [reply pkg/auth/setup/setup.go:189]: Uhh, no.  That was to remind me to refactor.  😊

**Review by arielshaqed — COMMENTED** [review]
THANKS!

No major changes, so I will pull once tests pass.

**Pull request merged**